docs/content/docs/testing/tinylicious.md

```diff
@@ -1,17 +1,98 @@
 ---
 title: Tinylicious
 menuPosition: 1
-status: unwritten
+editor: sdeshpande3
 ---
 
 ## What is Tinylicious?
 
-Our local service used for prototyping
-
-Link to repo
+Tinylicious is a local, in-memory Fluid service intended for prototyping and development purpose. You can use both [AzureClient]]({{< relref "azureclient.md" >}}) and [TinyliciousClient]({{< relref "tinyliciousclient.md" >}}) with Tinylicious for testing purposes.
 
 ## Using Tinylicious locally
 
+You can run Tinylicious locally by executing the following command:
+
+```sh
+npx tinylicious@latest
+```
+
+By default, Tinylicious runs on port `7070`. You can change port by setting the `PORT` environment variable when running Tinylicious. Setting environment variables will vary based on the shell you are using. For example, the `Windows PowerShell` commands below will run Tinylicious on port `6502`.
+
+```sh
+$env:PORT=6502
+npx tinylicious@latest
+```
+
+Now Tinylicious is listening on port `6502`.
+
 ## How to deploy using Tinylicious
 
-## Using custom domains with Tinylicious
+The `AzureClient` supports Tinylicious for development purposes in addition to deployed Azure Fluid Relay service instances for production scenarios.
```

Comment:
As we talked about let's get this to focus on TinyliciousClient and have a second doc on DCOM about AzureClient